apps/admin/backend/src/app.ts

```diff
@@ -692,7 +693,7 @@ function buildApi({
       groupBy: Tabulation.GroupBy;
     }): Array<Tabulation.GroupOf<Tabulation.CardCounts>> {
       const electionId = loadCurrentElectionIdOrThrow(workspace);
-      return Object.values(
+      return groupMapToGroupList(
```

Comment:
Now, when converting at the boundaries of the tabulation code, you have to use `groupMapToGroupList` rather than `Object.values`. The typing should help encourage this, because `GroupList` items have attributes like `precinctId`, `partyId`, etc. while the `GroupMap` items do not, because they assume that data is encoded in the key.